src/layouts/AbsoluteLayout.vue

```diff
@@ -1,23 +1,63 @@
 <template>
-    <div class="container">
+    <div class="nvw-absolute-layout" :style="{width: convertPxStyle(width), height: convertPxStyle(height), backgroundColor: backgroundColor}">
         <slot></slot>
     </div>
 </template>
 
 <script>
 export default {
   name: 'AbsoluteLayout',
+  props: {
+    width: {
+      default: '100%',
+    },
+    height: {
+      default: '100%',
+    },
+    backgroundColor: {},
+  },
+  mounted() {
+    this.$slots.default.forEach(vnode => {
+      const { propsData } = vnode.componentOptions.propsData;
+
+      for (const prop in propsData) {
+        if (!propsData.hasOwnProperty(prop)) continue;
+
+        switch (prop) {
+          case 'backgroundColor':
+            vnode.elm.style.backgroundColor = propsData[prop];
+            break;
+          case 'top':
```

Comment:
@ozercevikaslan we don't need these props anymore do we? please replace with common directive

Reply:
This comment will be a general reply to the issues in the components we did not consider to develop in this sprint. These components have not been updated yet in purpose not to lose time.